"mvn -N io.takari:maven:wrapper" is required to provision keycloak-benchmark in minikube on M1

### Describe the bug

When running the task file, the installation fails to run task "tlsdisableagent"(seems like " mvn -N io.takari:maven:wrapper" is missing, install the wrapper then the installation works):


```

annewang@Annes-MBP minikube % task
task: [keycloak-cli-download] mkdir -p keycloak-cli
task: [keycloak-cli-download] rm -f keycloak-cli/*.zip
task: [env] mkdir -p .task
task: [keycloak-cli-download] curl -L -f https://github.com/keycloak/keycloak/releases/download/nightly/keycloak-999-SNAPSHOT.zip -o keycloak-cli/keycloak.zip || curl -L -f https://github.com/keycloak/keycloak/releases/download/19.0.1/keycloak-19.0.1.zip -o keycloak-cli/keycloak.zip

task: [env] echo 168h > .task/var-KB_RETENTION
task: [env] echo 5 > .task/var-KC_DB_POOL_INITIAL_SIZE
task: [env] echo 10 > .task/var-KC_DB_POOL_MAX_SIZE
task: [env] echo 5 > .task/var-KC_DB_POOL_MIN_SIZE
task: [env] echo  > .task/var-KC_CONTAINER_IMAGE
task: [env] echo  > .task/var-KC_STORAGE
task: [env] echo postgres > .task/var-KC_DATABASE
[keycloak-cli-download]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[keycloak-cli-download]                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
task: [ipchange] rm -f .task/checksum/keycloak
task: [tlsdisableagent] bash -c ./buildtlsagent.sh
task: [ipchange] rm -f .task/checksum/prometheus
task: [dataset] ./mvnw -B -am -pl dataset clean install -DskipTests
task: [ipchange] rm -f .task/checksum/monitoring
task: [ipchange] mkdir -p .task
task: [ipchange] echo -n 10.0.2.15 > .task/status-ipchange
task: [prometheus] kubectl create namespace monitoring || true
[prometheus] namespace/monitoring created
task: [prometheus] helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
[dataset] --2023-01-16 13:38:52--  https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.1.0/maven-wrapper-3.1.0.jar
[prometheus] "prometheus-community" already exists with the same configuration, skipping
task: [prometheus] helm repo update
[prometheus] Hang tight while we grab the latest from your chart repositories...
[dataset] Resolving repo.maven.apache.org (repo.maven.apache.org)... 146.75.92.215
[tlsdisableagent] + cd java-instrumentation-tool
[tlsdisableagent] + ../../../mvnw clean package
[prometheus] ...Successfully got an update from the "codecentric" chart repository
[dataset] Connecting to repo.maven.apache.org (repo.maven.apache.org)|146.75.92.215|:443... connected.
[tlsdisableagent] Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
[tlsdisableagent] Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
  0  162M    0  127k    0     0    98k      0  0:27:56  0:00:01  0:27:55   98k
[dataset] HTTP request sent, awaiting response... 200 OK
[dataset] Length: 58727 (57K) [application/java-archive]
[dataset] Saving to: ‘/Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/.mvn/wrapper/maven-wrapper.jar’
[dataset] 
[dataset]      0K .......... .......... .......... .......... .......... 87%  272K 0s
[dataset]     50K .......                                               100%  472K=0.2s
[dataset] 
[dataset] 2023-01-16 13:38:54 (288 KB/s) - ‘/Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/.mvn/wrapper/maven-wrapper.jar’ saved [58727/58727]
[dataset] 
[dataset] [INFO] Scanning for projects...
[dataset] [INFO] ------------------------------------------------------------------------
[dataset] [INFO] Reactor Build Order:
[dataset] [INFO] 
[dataset] [INFO] Keycloak Benchmark Parent                                          [pom]
[dataset] [INFO] keycloak-benchmark-dataset                                         [jar]
[dataset] [INFO] 
[dataset] [INFO] ---------------< org.keycloak:keycloak-benchmark-parent >---------------
[dataset] [INFO] Building Keycloak Benchmark Parent 0.9-SNAPSHOT                    [1/2]
[dataset] [INFO] --------------------------------[ pom ]---------------------------------
[dataset] [INFO] 
[dataset] [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ keycloak-benchmark-parent ---
[dataset] [INFO] 
[dataset] [INFO] --- maven-install-plugin:2.4:install (default-install) @ keycloak-benchmark-parent ---
[dataset] [INFO] Installing /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/pom.xml to /Users/annewang/.m2/repository/org/keycloak/keycloak-benchmark-parent/0.9-SNAPSHOT/keycloak-benchmark-parent-0.9-SNAPSHOT.pom
[dataset] [INFO] 
[dataset] [INFO] --------------< org.keycloak:keycloak-benchmark-dataset >---------------
[dataset] [INFO] Building keycloak-benchmark-dataset 0.9-SNAPSHOT                   [2/2]
[dataset] [INFO] --------------------------------[ jar ]---------------------------------
[dataset] [WARNING] The artifact org.apache.commons:commons-io:jar:1.3.2 has been relocated to commons-io:commons-io:jar:1.3.2: https://issues.sonatype.org/browse/MVNCENTRAL-244
[dataset] [INFO] 
[dataset] [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ keycloak-benchmark-dataset ---
[dataset] [INFO] 
[dataset] [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[dataset] [INFO] Copying 2 resources
[dataset] [INFO] 
[dataset] [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Changes detected - recompiling the module!
[dataset] [INFO] Compiling 20 source files to /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/target/classes
[dataset] [INFO] 
[dataset] [INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[dataset] [INFO] skip non existing resourceDirectory /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/src/test/resources
[dataset] [INFO] 
[dataset] [INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Changes detected - recompiling the module!
[dataset] [INFO] Compiling 3 source files to /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/target/test-classes
[dataset] [INFO] 
[dataset] [INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Tests are skipped.
[dataset] [INFO] 
[dataset] [INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Building jar: /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/target/keycloak-benchmark-dataset-0.9-SNAPSHOT.jar
[dataset] [INFO] 
[dataset] [INFO] --- maven-dependency-plugin:3.3.0:unpack (unpack) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Configured Artifact: org.keycloak:keycloak-quarkus-dist:999-SNAPSHOT:zip
[dataset] [INFO] Unpacking /Users/annewang/.m2/repository/org/keycloak/keycloak-quarkus-dist/999-SNAPSHOT/keycloak-quarkus-dist-999-SNAPSHOT.zip to /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/target/keycloak with includes "" and excludes ""
[dataset] [INFO] 
[dataset] [INFO] --- maven-failsafe-plugin:3.0.0-M7:integration-test (default) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Tests are skipped.
[dataset] [INFO] 
[dataset] [INFO] --- maven-failsafe-plugin:3.0.0-M7:verify (default) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Tests are skipped.
[dataset] [INFO] 
[dataset] [INFO] --- maven-install-plugin:2.4:install (default-install) @ keycloak-benchmark-dataset ---
[dataset] [INFO] Installing /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/target/keycloak-benchmark-dataset-0.9-SNAPSHOT.jar to /Users/annewang/.m2/repository/org/keycloak/keycloak-benchmark-dataset/0.9-SNAPSHOT/keycloak-benchmark-dataset-0.9-SNAPSHOT.jar
[dataset] [INFO] Installing /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/dataset/pom.xml to /Users/annewang/.m2/repository/org/keycloak/keycloak-benchmark-dataset/0.9-SNAPSHOT/keycloak-benchmark-dataset-0.9-SNAPSHOT.pom
[dataset] [INFO] ------------------------------------------------------------------------
[dataset] [INFO] Reactor Summary for Keycloak Benchmark Parent 0.9-SNAPSHOT:
[dataset] [INFO] 
[dataset] [INFO] Keycloak Benchmark Parent .......................... SUCCESS [  0.121 s]
[dataset] [INFO] keycloak-benchmark-dataset ......................... SUCCESS [  3.649 s]
[dataset] [INFO] ------------------------------------------------------------------------
[dataset] [INFO] BUILD SUCCESS
[dataset] [INFO] ------------------------------------------------------------------------
[dataset] [INFO] Total time:  3.820 s
[dataset] [INFO] Finished at: 2023-01-16T13:38:58+13:00
[dataset] [INFO] ------------------------------------------------------------------------
task: Failed to run task "tlsdisableagent": exit status 1
```

Now install wrapper:

```
annewang@Annes-MBP minikube % mvn -N io.takari:maven:wrapper

[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/maven/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/maven/maven-metadata.xml (755 B at 704 B/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/maven/0.7.7/maven-0.7.7.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/maven/0.7.7/maven-0.7.7.pom (2.3 kB at 6.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/takari/27/takari-27.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/takari/27/takari-27.pom (14 kB at 20 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/maven/0.7.7/maven-0.7.7.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/maven/0.7.7/maven-0.7.7.jar (9.0 kB at 23 kB/s)
[INFO] 
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven:0.7.7:wrapper (default-cli) @ standalone-pom ---
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.pom (2.4 kB at 6.6 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/takari-archiver/0.1.9/takari-archiver-0.1.9.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/takari-archiver/0.1.9/takari-archiver-0.1.9.pom (1.8 kB at 5.1 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/takari/15/takari-15.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/takari/15/takari-15.pom (15 kB at 35 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/14.0.1/guava-14.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/14.0.1/guava-14.0.1.pom (5.4 kB at 14 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/14.0.1/guava-parent-14.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/14.0.1/guava-parent-14.0.1.pom (2.6 kB at 7.1 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.pom (11 kB at 28 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/33/commons-parent-33.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/33/commons-parent-33.pom (53 kB at 85 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.pom (3.1 kB at 8.7 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/graph/takari-graph/0.0.3/takari-graph-0.0.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/graph/takari-graph/0.0.3/takari-graph-0.0.3.pom (4.7 kB at 13 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/takari/14/takari-14.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/takari/14/takari-14.pom (13 kB at 33 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.16/plexus-utils-3.0.16.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.16/plexus-utils-3.0.16.pom (3.4 kB at 4.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints-core/2.6.0/blueprints-core-2.6.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints-core/2.6.0/blueprints-core-2.6.0.pom (3.6 kB at 9.4 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints/2.6.0/blueprints-2.6.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints/2.6.0/blueprints-2.6.0.pom (7.0 kB at 8.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.pom (5.4 kB at 14 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/11/oss-parent-11.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/11/oss-parent-11.pom (23 kB at 39 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.pom (1.3 kB at 3.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/10/oss-parent-10.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/10/oss-parent-10.pom (23 kB at 40 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.pom (6.0 kB at 15 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.2.3/maven-model-3.2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.2.3/maven-model-3.2.3.pom (4.1 kB at 11 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven/3.2.3/maven-3.2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven/3.2.3/maven-3.2.3.pom (23 kB at 42 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-parent/24/maven-parent-24.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-parent/24/maven-parent-24.pom (37 kB at 33 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/apache/14/apache-14.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/apache/14/apache-14.pom (15 kB at 22 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.17/plexus-utils-3.0.17.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.17/plexus-utils-3.0.17.pom (3.4 kB at 8.0 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/compiler/0.8.15/compiler-0.8.15.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/compiler/0.8.15/compiler-0.8.15.pom (5.6 kB at 10 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/mustache.java/0.8.15/mustache.java-0.8.15.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/mustache.java/0.8.15/mustache.java-0.8.15.pom (3.5 kB at 8.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/squareup/javapoet/1.0.0/javapoet-1.0.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/squareup/javapoet/1.0.0/javapoet-1.0.0.pom (3.5 kB at 8.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/antlr/antlr/2.7.7/antlr-2.7.7.pom
Downloaded from central: https://repo.maven.apache.org/maven2/antlr/antlr/2.7.7/antlr-2.7.7.pom (632 B at 1.7 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/airlift/airline/0.6/airline-0.6.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/airlift/airline/0.6/airline-0.6.pom (6.8 kB at 12 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/12.0/guava-12.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/12.0/guava-12.0.pom (5.3 kB at 5.4 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/12.0/guava-parent-12.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/12.0/guava-parent-12.0.pom (2.8 kB at 6.1 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/antlr/antlr4-runtime/4.5/antlr4-runtime-4.5.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/antlr/antlr4-runtime/4.5/antlr4-runtime-4.5.pom (2.7 kB at 6.2 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/abego/treelayout/org.abego.treelayout.core/1.0.1/org.abego.treelayout.core-1.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/abego/treelayout/org.abego.treelayout.core/1.0.1/org.abego.treelayout.core-1.0.1.pom (3.1 kB at 7.1 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-annotations/3.3/maven-plugin-annotations-3.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-annotations/3.3/maven-plugin-annotations-3.3.pom (1.6 kB at 4.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-tools/3.3/maven-plugin-tools-3.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-tools/3.3/maven-plugin-tools-3.3.pom (13 kB at 18 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/takari-archiver/0.1.9/takari-archiver-0.1.9.jar
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/14.0.1/guava-14.0.1.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/takari-archiver/0.1.9/takari-archiver-0.1.9.jar (41 kB at 43 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/takari/graph/takari-graph/0.0.3/takari-graph-0.0.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.jar (239 kB at 176 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints-core/2.6.0/blueprints-core-2.6.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar (51 kB at 30 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/14.0.1/guava-14.0.1.jar (2.2 MB at 1.2 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/tinkerpop/blueprints/blueprints-core/2.6.0/blueprints-core-2.6.0.jar (274 kB at 130 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.jar (33 kB at 15 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.2.3/maven-model-3.2.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/takari/graph/takari-graph/0.0.3/takari-graph-0.0.3.jar (173 kB at 78 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/compiler/0.8.15/compiler-0.8.15.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.2.3/maven-model-3.2.3.jar (160 kB at 62 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.jar (193 kB at 69 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/antlr/antlr/2.7.7/antlr-2.7.7.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar (65 kB at 22 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/airlift/airline/0.6/airline-0.6.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/airlift/airline/0.6/airline-0.6.jar (87 kB at 26 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/antlr/antlr4-runtime/4.5/antlr4-runtime-4.5.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/spullara/mustache/java/compiler/0.8.15/compiler-0.8.15.jar (116 kB at 34 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/abego/treelayout/org.abego.treelayout.core/1.0.1/org.abego.treelayout.core-1.0.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar (366 kB at 106 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-annotations/3.3/maven-plugin-annotations-3.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/antlr/antlr4-runtime/4.5/antlr4-runtime-4.5.jar (374 kB at 97 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-annotations/3.3/maven-plugin-annotations-3.3.jar (14 kB at 3.6 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/antlr/antlr/2.7.7/antlr-2.7.7.jar (445 kB at 109 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/abego/treelayout/org.abego.treelayout.core/1.0.1/org.abego.treelayout.core-1.0.1.jar (26 kB at 6.1 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.jar (866 kB at 73 kB/s)
[INFO] 
[INFO] Maven Wrapper version 0.5.6 has been successfully set up for your project.
[INFO] Using Apache Maven: 3.6.3
[INFO] Repo URL in properties file: https://repo.maven.apache.org/maven2
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  32.876 s
[INFO] Finished at: 2023-01-16T13:41:20+13:00
[INFO] ------------------------------------------------------------------------
annewang@Annes-MBP minikube % 
annewang@Annes-MBP minikube % task                          
task: Task "split" is up to date
task: [ipchange] rm -f .task/checksum/keycloak
task: [keycloak-cli-download] mkdir -p keycloak-cli
task: [keycloak-cli-download] rm -f keycloak-cli/*.zip
task: [tlsdisableagent] bash -c ./buildtlsagent.sh
task: [ipchange] rm -f .task/checksum/prometheus
task: [keycloak-cli-download] curl -L -f https://github.com/keycloak/keycloak/releases/download/nightly/keycloak-999-SNAPSHOT.zip -o keycloak-cli/keycloak.zip || curl -L -f https://github.com/keycloak/keycloak/releases/download/19.0.1/keycloak-19.0.1.zip -o keycloak-cli/keycloak.zip

task: [ipchange] rm -f .task/checksum/monitoring
[tlsdisableagent] + cd java-instrumentation-tool
[tlsdisableagent] + ../../../mvnw clean package
task: [ipchange] mkdir -p .task
[keycloak-cli-download]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[keycloak-cli-download]                                  Dload  Upload   Total   Spent    Left  Speed
task: [env] mkdir -p .task
task: [ipchange] echo -n 10.0.2.15 > .task/status-ipchange
task: [env] echo 168h > .task/var-KB_RETENTION
task: [env] echo 5 > .task/var-KC_DB_POOL_INITIAL_SIZE
task: [env] echo 10 > .task/var-KC_DB_POOL_MAX_SIZE
task: [env] echo 5 > .task/var-KC_DB_POOL_MIN_SIZE
task: [env] echo  > .task/var-KC_CONTAINER_IMAGE
task: [env] echo  > .task/var-KC_STORAGE
task: Task "dataset" is up to date
task: [env] echo postgres > .task/var-KC_DATABASE
task: [datasetprovider] mkdir -p keycloak/providers
task: [prometheus] kubectl create namespace monitoring || true
task: [datasetprovider] rm -f keycloak/providers/keycloak-benchmark-dataset-*.jar
task: [datasetprovider] cp ../../dataset/target/keycloak-benchmark-dataset-*.jar keycloak/providers
[prometheus] Error from server (AlreadyExists): namespaces "monitoring" already exists
task: [prometheus] helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
[prometheus] "prometheus-community" already exists with the same configuration, skipping
task: [prometheus] helm repo update
[prometheus] Hang tight while we grab the latest from your chart repositories...
[prometheus] ...Successfully got an update from the "codecentric" chart repository
[prometheus] ...Successfully got an update from the "jaegertracing" chart repository
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[tlsdisableagent] [INFO] Scanning for projects...
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] ---------------< de.alech.jit:java-instrumentation-tool >---------------
[tlsdisableagent] [INFO] Building java-instrumentation-tool 0.1
[tlsdisableagent] [INFO] --------------------------------[ jar ]---------------------------------
[prometheus] ...Successfully got an update from the "grafana" chart repository
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[tlsdisableagent] [INFO] skip non existing resourceDirectory /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/src/main/resources
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] Changes detected - recompiling the module!
[tlsdisableagent] [INFO] Compiling 5 source files to /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/target/classes
[prometheus] ...Successfully got an update from the "prometheus-community" chart repository
[tlsdisableagent] [WARNING] /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/src/main/java/de/alech/jit/agent/JavaAgent.java: /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/src/main/java/de/alech/jit/agent/JavaAgent.java uses unchecked or unsafe operations.
[tlsdisableagent] [WARNING] /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/src/main/java/de/alech/jit/agent/JavaAgent.java: Recompile with -Xlint:unchecked for details.
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[tlsdisableagent] [INFO] skip non existing resourceDirectory /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/src/test/resources
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] No sources to compile
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] No tests to run.
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] Building jar: /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/target/java-instrumentation-tool-0.1.jar
[tlsdisableagent] [INFO] 
[tlsdisableagent] [INFO] --- maven-shade-plugin:3.2.1:shade (default) @ java-instrumentation-tool ---
[tlsdisableagent] [INFO] Including org.javassist:javassist:jar:3.25.0-GA in the shaded jar.
[tlsdisableagent] [INFO] Replacing original artifact with shaded artifact.
[tlsdisableagent] [INFO] Replacing /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/target/java-instrumentation-tool-0.1.jar with /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/target/java-instrumentation-tool-0.1-shaded.jar
[tlsdisableagent] [INFO] Dependency-reduced POM written at: /Users/annewang/Documents/s6/project/aws-h1/kinetik-tile-helm/resources/keycloak-benchmark/provision/tlsdisableagent/java-instrumentation-tool/dependency-reduced-pom.xml
[tlsdisableagent] [INFO] ------------------------------------------------------------------------
[tlsdisableagent] [INFO] BUILD SUCCESS
[tlsdisableagent] [INFO] ------------------------------------------------------------------------
[tlsdisableagent] [INFO] Total time:  0.875 s
[tlsdisableagent] [INFO] Finished at: 2023-01-16T13:41:27+13:00
[tlsdisableagent] [INFO] ------------------------------------------------------------------------
[tlsdisableagent] + cd ..
[tlsdisableagent] + FILE=tlscheckdisable-agent.jar
[tlsdisableagent] + /Library/Java/JavaVirtualMachines/temurin-19.jdk/Contents/Home/bin/java -jar java-instrumentation-tool/target/java-instrumentation-tool-0.1.jar tlscheckdisable.txt tlscheckdisable-agent.jar
[tlsdisableagent] + echo agent: tlscheckdisable-agent.jar
[tlsdisableagent] agent: tlscheckdisable-agent.jar
[prometheus] ...Successfully got an update from the "my-repo" chart repository
[prometheus] Update Complete. ⎈Happy Helming!⎈
task: [prometheus] helm upgrade --install prometheus prometheus-community/kube-prometheus-stack --version 39.11.0 -f monitoring.yaml --set grafana."grafana\.ini".server.root_url=https://grafana.10.0.2.15.nip.io --set prometheus.prometheusSpec.retention=168h
[prometheus] Release "prometheus" does not exist. Installing it now.
[prometheus] NAME: prometheus
[prometheus] LAST DEPLOYED: Mon Jan 16 13:41:30 2023
[prometheus] NAMESPACE: default
[prometheus] STATUS: deployed
[prometheus] REVISION: 1
[prometheus] NOTES:
[prometheus] kube-prometheus-stack has been installed. Check its status by running:
[prometheus]   kubectl --namespace monitoring get pods -l "release=prometheus"
[prometheus] 
[prometheus] Visit https://github.com/prometheus-operator/kube-prometheus for instructions on how to create & configure Alertmanager and Prometheus instances using the Operator.
task: [tempo] helm delete tempo -n monitoring || exit 0
task: [loki] helm repo add grafana https://grafana.github.io/helm-charts
task: [jaeger] helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
task: [promtail] helm repo add grafana https://grafana.github.io/helm-charts
task: [monitoring] helm upgrade --install monitoring --set hostname=10.0.2.15.nip.io monitoring
[promtail] "grafana" already exists with the same configuration, skipping
[jaeger] "jaegertracing" already exists with the same configuration, skipping
task: [promtail] helm repo update
task: [jaeger] helm repo update
[tempo] Error: uninstall: Release not loaded: tempo: release: not found
[monitoring] Release "monitoring" does not exist. Installing it now.
[jaeger] Hang tight while we grab the latest from your chart repositories...
[promtail] Hang tight while we grab the latest from your chart repositories...
[jaeger] ...Successfully got an update from the "codecentric" chart repository
[promtail] ...Successfully got an update from the "codecentric" chart repository
[jaeger] ...Successfully got an update from the "jaegertracing" chart repository
[promtail] ...Successfully got an update from the "jaegertracing" chart repository
[monitoring] NAME: monitoring
[monitoring] LAST DEPLOYED: Mon Jan 16 13:42:04 2023
[monitoring] NAMESPACE: default
[monitoring] STATUS: deployed
[monitoring] REVISION: 1
[monitoring] TEST SUITE: None
[monitoring] NOTES:
[monitoring] Grafana will be available on https://grafana.10.0.2.15.nip.io
[monitoring] Prometheus will be available on https://prometheus.10.0.2.15.nip.io
[monitoring] kubebox will be available on https://kubebox.10.0.2.15.nip.io
[monitoring] jaeger will be available on https://jaeger.10.0.2.15.nip.io
[monitoring] Graphite metrics can be sent to 10.0.2.15.nip.io:30007
[monitoring] Jaeger Traces can be sent to 10.0.2.15.nip.io:30008
[loki] "grafana" already exists with the same configuration, skipping
task: [loki] helm repo update
[loki] Hang tight while we grab the latest from your chart repositories...
[jaeger] ...Successfully got an update from the "grafana" chart repository
[promtail] ...Successfully got an update from the "grafana" chart repository
[loki] ...Successfully got an update from the "codecentric" chart repository
[loki] ...Successfully got an update from the "jaegertracing" chart repository
[promtail] ...Successfully got an update from the "prometheus-community" chart repository
[jaeger] ...Successfully got an update from the "prometheus-community" chart repository
[loki] ...Successfully got an update from the "grafana" chart repository
[jaeger] ...Successfully got an update from the "my-repo" chart repository
[jaeger] Update Complete. ⎈Happy Helming!⎈
task: [jaeger] helm upgrade --install jaeger jaegertracing/jaeger --version 0.58.0 -n monitoring -f jaeger/values.yaml
[jaeger] Release "jaeger" does not exist. Installing it now.
[promtail] ...Successfully got an update from the "my-repo" chart repository
[promtail] Update Complete. ⎈Happy Helming!⎈
task: [promtail] helm upgrade --install promtail grafana/promtail --version 6.3.0 -n monitoring -f promtail.yaml
[promtail] Release "promtail" does not exist. Installing it now.
[loki] ...Successfully got an update from the "my-repo" chart repository
[loki] ...Successfully got an update from the "prometheus-community" chart repository
[loki] Update Complete. ⎈Happy Helming!⎈
task: [loki] bash -c "helm upgrade --install loki grafana/loki --version 3.0.0 -n monitoring -f loki.yaml --set config.table_manager.retention_period=168h || (helm delete loki -n monitoring && helm upgrade --install loki grafana/loki --version 3.0.0 -n monitoring -f loki.yaml --set config.table_manager.retention_period=168h)"

[loki] Release "loki" does not exist. Installing it now.
[promtail] NAME: promtail
[promtail] LAST DEPLOYED: Mon Jan 16 13:42:10 2023
[promtail] NAMESPACE: monitoring
[promtail] STATUS: deployed
[promtail] REVISION: 1
[promtail] TEST SUITE: None
[promtail] NOTES:
[promtail] ***********************************************************************
[promtail]  Welcome to Grafana Promtail
[promtail]  Chart version: 6.3.0
[promtail]  Promtail version: 2.6.1
[promtail] ***********************************************************************
[promtail] 
[promtail] Verify the application is working by running these commands:
[promtail] * kubectl --namespace monitoring port-forward daemonset/promtail 3101
[promtail] * curl http://127.0.0.1:3101/metrics
[loki] NAME: loki
[loki] LAST DEPLOYED: Mon Jan 16 13:42:13 2023
[loki] NAMESPACE: monitoring
[loki] STATUS: deployed
[loki] REVISION: 1
[loki] NOTES:
[loki] ***********************************************************************
[loki]  Welcome to Grafana Loki
[loki]  Chart version: 3.0.0
[loki]  Loki version: 2.6.1
[loki] ***********************************************************************
[loki] 
[loki] Installed components:
[loki] * loki
[jaeger] NAME: jaeger
[jaeger] LAST DEPLOYED: Mon Jan 16 13:42:16 2023
[jaeger] NAMESPACE: monitoring
[jaeger] STATUS: deployed
[jaeger] REVISION: 1
[jaeger] TEST SUITE: None
[jaeger] NOTES:
[jaeger] ###################################################################
[jaeger] ### IMPORTANT: Ensure that storage is explicitly configured     ###
[jaeger] ### Default storage options are subject to change.              ###
[jaeger] ###                                                             ###
[jaeger] ### IMPORTANT: The use of <component>.env: {...} is deprecated. ###
[jaeger] ### Please use <component>.extraEnv: [] instead.                ###
[jaeger] ###################################################################
[jaeger] 
[jaeger] You can log into the Jaeger Query UI here:
[jaeger] 
[jaeger]   export POD_NAME=$(kubectl get pods --namespace monitoring -l "app.kubernetes.io/instance=jaeger,app.kubernetes.io/component=query" -o jsonpath="{.items[0].metadata.name}")
[jaeger]   echo http://127.0.0.1:8080/
[jaeger]   kubectl port-forward --namespace monitoring $POD_NAME 8080:16686
task: [jaeger] envsubst < jaeger/deployment-patch.yaml > .task/subtask-jaeger-patchfile.yaml
task: [jaeger] echo $KB_RETENTION
[jaeger] 168h
task: [jaeger] kubectl patch deployment jaeger -n monitoring --patch-file .task/subtask-jaeger-patchfile.yaml
[jaeger] deployment.apps/jaeger patched
task: [keycloak] kubectl create namespace keycloak || true
[keycloak] namespace/keycloak created
task: [keycloak] kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/keycloaks.k8s.keycloak.org-v1.yml
[keycloak] customresourcedefinition.apiextensions.k8s.io/keycloaks.k8s.keycloak.org created
task: [keycloak] kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/keycloakrealmimports.k8s.keycloak.org-v1.yml
[keycloak] customresourcedefinition.apiextensions.k8s.io/keycloakrealmimports.k8s.keycloak.org created
task: [keycloak] kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/kubernetes.yml || (kubectl -n keycloak delete deployment/keycloak-operator && kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/kubernetes.yml)
[keycloak] serviceaccount/keycloak-operator created
[keycloak] service/keycloak-operator created
[keycloak] role.rbac.authorization.k8s.io/keycloak-operator-role created
[keycloak] rolebinding.rbac.authorization.k8s.io/keycloak-operator-role-binding created
[keycloak] rolebinding.rbac.authorization.k8s.io/keycloak-operator-view created
[keycloak] rolebinding.rbac.authorization.k8s.io/keycloakcontroller-role-binding created
[keycloak] rolebinding.rbac.authorization.k8s.io/keycloakrealmimportcontroller-role-binding created
[keycloak] clusterrole.rbac.authorization.k8s.io/keycloakcontroller-cluster-role created
[keycloak] clusterrole.rbac.authorization.k8s.io/keycloakrealmimportcontroller-cluster-role created
[keycloak] deployment.apps/keycloak-operator created
task: [keycloak] helm upgrade --install keycloak --set hostname=10.0.2.15.nip.io --set dbPoolInitialSize=5 --set dbPoolMinSize=5 --set dbPoolMaxSize=10 --set storage= --set database=postgres --set keycloakImage= keycloak

[keycloak] Release "keycloak" does not exist. Installing it now.
[keycloak] NAME: keycloak
[keycloak] LAST DEPLOYED: Mon Jan 16 13:42:20 2023
[keycloak] NAMESPACE: default
[keycloak] STATUS: deployed
[keycloak] REVISION: 1
[keycloak] TEST SUITE: None
[keycloak] NOTES:
[keycloak] Keycloak will be available on https://keycloak.10.0.2.15.nip.io
[keycloak]   user: admin
[keycloak]   password: admin
[keycloak] sqlpad will be available on http://sqlpad.10.0.2.15.nip.io
[keycloak]   user: admin
[keycloak]   password: admin
[keycloak] Cryostat will be available on https://cryostat.10.0.2.15.nip.io
[keycloak] Connect to PostgreSQL on 10.0.2.15.nip.io:30009
[keycloak]   user: keycloak
[keycloak]   password: pass
[keycloak]   JDBC URL: jdbc:postgresql://10.0.2.15.nip.io:30009/keycloak
task: [keycloak] bash -c ' if [ "postgres" == "cockroach-operator" ]; then kubectl -n keycloak -o yaml get crdbclusters.crdb.cockroachlabs.com/cockroach > .task/status-keycloak-db.json; elif [ "postgres" == "cockroach-single" ]; then kubectl get deployment/cockroach -n keycloak -o=jsonpath="{.spec}" > .task/status-keycloak-db.json; elif [ "postgres" != "none" ]; then kubectl get deployment/postgres -n keycloak -o=jsonpath="{.spec}" > .task/status-keycloak-db.json; else echo "none" > .task/status-keycloak-db.json; fi'

task: [keycloak] bash -c 'kubectl get pods -A | grep -E "(BackOff|Error)" | tr -s " " | cut -d" " -f1-2 | xargs -r -L 1 kubectl delete pod -n'
task: [keycloak] bash -c 'sleep 1'
task: [keycloak] ./init-database.sh postgres
task: [keycloak] ./isup.sh
100  162M  100  162M    0     0  1406k      0  0:01:58  0:01:58 --:--:-- 1161k
task: [keycloak-cli-unzip] rm -rf keycloak-cli/keycloak-999-SNAPSHOT keycloak-cli/keycloak-19.0.1
task: [keycloak-cli-unzip] rm -rf keycloak-cli/keycloak
task: [keycloak-cli-unzip] unzip -o -q keycloak-cli/keycloak.zip -d keycloak-cli
task: [keycloak-cli-unzip] mv keycloak-cli/keycloak-999-SNAPSHOT keycloak-cli/keycloak || mv keycloak-cli/keycloak-19.0.1 keycloak-cli/keycloak

```

### Version

latest

### Expected behavior

Installation works without failure

### Actual behavior

Fails, then run the following command to continue the installation:
mvn -N io.takari:maven:wrapper


### How to Reproduce?

Install on apple M1 and provision with minikube.

### Anything else?

N/a

--- a/provision/minikube/Taskfile.yaml
+++ b/provision/minikube/Taskfile.yaml
@@ -184,9 +184,22 @@ tasks:
       - promtail.yaml
       - .task/subtask-{{.TASK}}.yaml
 
+  mvnw:
+    dir: ../..
+    run: once
+    cmds:
+      # download maven wrapper to avoid concurrent downloads with can break the build
+      - ./mvnw -v
+    sources:
+      - ../../.mvn/wrapper/maven-wrapper.properties
+      - .task/subtask-{{.TASK}}.yaml
+    generates:
+      - ../../.mvn/wrapper/maven-wrapper.jar
+
   dataset:
     deps:
       - split
+      - mvnw
     dir: ../..
     cmds:
       - ./mvnw -B -am -pl dataset clean install -DskipTests
@@ -222,6 +235,7 @@ tasks:
   tlsdisableagent:
     deps:
       - split
+      - mvnw
     dir: ../tlsdisableagent
     cmds:
       - bash -c ./buildtlsagent.sh

--- a/provision/tlsdisableagent/buildtlsagent.sh
+++ b/provision/tlsdisableagent/buildtlsagent.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euxo pipefail
 cd java-instrumentation-tool
-../../../mvnw clean package
+../../../mvnw clean package -B
 cd ..
 FILE=tlscheckdisable-agent.jar
 $JAVA_HOME/bin/java -jar java-instrumentation-tool/target/java-instrumentation-tool-*.jar \

